Gds/src/fprime_gds/executables/fprime_cli.py

```diff
@@ -212,7 +216,8 @@ def create_subparser(cls, parent_parser: argparse.ArgumentParser):
         """
         channels_parser = parent_parser.add_parser(
             "channels",
-            description="print out new telemetry data that has been received from the F Prime instance, sorted by timestamp",
+            description="print out new telemetry data that has been received from"
+                        "the F Prime instance, sorted by timestamp",
```

Comment:
Same as above. I prefer to not separate the strings into two quotes on separate line. Might be better to revert it back to what they were.